lib/services/local/presence.go

```diff
@@ -1695,6 +1695,9 @@ func (s *PresenceService) listResources(ctx context.Context, req proto.ListResou
 	case types.KindWindowsDesktopService:
 		keyPrefix = []string{windowsDesktopServicesPrefix}
 		unmarshalItemFunc = backendItemToWindowsDesktopService
+	case types.KindWindowsDesktop:
+		keyPrefix = []string{windowsDesktopsPrefix}
+		unmarshalItemFunc = backendItemToWindowsDesktop
```

Comment:
Not sure if this is right, WindowsDesktop isnt part of/managed by the presence service, I'm not sure if that matters, but It makes the test look a little funny